server/controller/user.js

```diff
@@ -5,6 +5,49 @@ import UserModel from '../models/models';
 
 
 const User = {
+  async create(req, res) {
+    if (req.body.password !== req.body.confirmPassword) {
+      res.status(400).json({
+        status: 400,
+        error: 'Passwords do not match',
+      });
+    }
+
+    const user = new UserModel(req.body);
```

Comment:
This variable name user is ambiguous. What kind of user? Providing more context in variable names helps improve code readability.

Reply:
Ok, I would I would make my variables comprehensive